tests/components/min_max/test_sensor.py

```diff
@@ -193,6 +228,7 @@ def test_not_enough_sensor_value(self):
         assert state.attributes.get("min_value") is None
         assert state.attributes.get("max_entity_id") is None
         assert state.attributes.get("max_value") is None
+        assert state.attributes.get("median") is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)